src/test/scheduler_tests.cpp

```diff
@@ -58,7 +56,7 @@ BOOST_AUTO_TEST_CASE(manythreads)
 
     boost::mutex counterMutex[10];
     int counter[10] = { 0 };
-    boost::random::mt19937 rng(insecure_rand());
+    boost::random::mt19937 rng(42);
```

Comment:
Hah. It's the result of a certain long-running computation to answer a question that I can't quite remember anymore but seemed ultimately important.